stream_alert_cli/test.py

```diff
@@ -53,6 +53,33 @@ def report_output(cols, failed):
 
     print '\t{}\ttest ({}): {}'.format(status, *cols)
 
+def report_output_summary(rules_fail_pass):
+    """Helper function to print the summary results of all tests
+    Args:
+        rules_fail_pass [list]: A list containing two lists for failed and passed
+            rule tests. The sublists contain tuples made up of: (rule_name, rule_description)
+    """
+    failed_tests = len(rules_fail_pass[0])
```

Comment:
I was thinking catching errors [here](https://github.com/airbnb/streamalert/blob/1fbd2f99a970df31e3a4bb4f3a0bccfd8fc340e6/stream_alert_cli/test.py#L261-L262) and [here](https://github.com/airbnb/streamalert/blob/1fbd2f99a970df31e3a4bb4f3a0bccfd8fc340e6/stream_alert_cli/test.py#L267-L268).  I think it would increase the clarity of the result since tests never actually run in those cases